tests/behat/notification.feature

```diff
@@ -0,0 +1,28 @@
+@local @local_edit_course_notification
+Feature: Edit notification
+  In order to show notification text
+  As an editor
+  I need to see a notification text when editing a course if it's set
+
+  Background:
+    Given the following "courses" exist:
+      | fullname | shortname | format |
+      | Course 1 | C1        | topics |
+    And I log in as "admin"
+    And I am on "Course 1" course homepage
+    And I set the following administration settings values:
+      | notificationstatement | Notification test text |
+
+  Scenario: Course edit mode off
```

Comment:
can you please add a couple of more scenarios to cover changing of a notification type setting?

Reply:
Also would be good to test if notification is empty, then we won't see it on the page

Reply:
Also would be good to test if we don't see the notification on other course pages in editing mode

Reply:
More cases have been added for notification types and empty check.
We can see the notification on the other course pages when editing on, because it's global setting for all courses.